study-builder/tenants/cmi/login_page.html

```diff
@@ -82,19 +80,23 @@
     var loginUrl = config.callbackURL + '/login';
 
     var registerUrl = '';
-    if (study_guid.toUpperCase() === angio) {
+    if (study_guid.toLowerCase() === 'angio') {
         // arz fixme to read from client var
         registerUrl = 'https://dev.ascproject.org/count-me-in';
-        // todo arz move to gcp bucket
-        headerLogo = 'https://storage.googleapis.com/angio-assets/angiosarcoma-project-isotype.png';
+        headerLogo = 'https://storage.googleapis.com/cmi-study-dev-assets/angio/angiosarcoma-project-isotype.png';
```

Comment:
Isn't prod login-page going to use dev-assets google bucket !!

Reply:
Right now, updating auth0 config across envs is a manual process. As part of that, we change URLs according to the env. So I'm checking in the dev urls into source and when we put into different env we can URL when we copy into auth0. For example, the `registerUrl` above will be changed to `staging.ascproject` when we go to staging. Likewise for `cmi-study-dev-assets` -> `cmi-study-assets` when we go to prod.